templates/Containerisation/helm/helm-install.yaml

```diff
@@ -0,0 +1,112 @@
+parameters:
```

Comment:
Can we prefix/group the parameters e.g. aks vars, dns vars, pr vars, github vars
could make it easier to identity what they are for

Reply:
like this? I wasn't sure if you meant using groups or just logically separating them. Could also prefix them too if you think it'd be handier?

Reply:
that will do.